hane-syntax/src/lower.rs

```diff
@@ -63,6 +80,60 @@ impl Command {
                 let ttype = ttype.lower(global, &mut names)?;
                 global.insert(name.clone());
                 LoweredCommandVariant::Axiom(name, ttype)
+            },
+            CommandVariant::Inductive(mut bodies) => {
+                let params = std::mem::take(&mut bodies[0].params);
+                bodies.iter()
```

Comment:
I'm not sure what is going on here,  are "bodies" constructors?

Reply:
A `body` here is one of the types along with it's constructors. I couldn't think of a better name